docs/website/docs/user-guides/deploy/_editing_devfile.mdx

```diff
@@ -0,0 +1,177 @@
+import Tabs from '@theme/Tabs';
```

Comment:
If it's not too out of the scope for this PR, can you add a section of what the final devfile would look like? I think it would help temporarily fix the problem with YAML.

Reply:
Would be outside the scope of this PR because of changes made upstream (devfile registry) would be difficult to sync between the two if there are any changes.

I'd have to find a better way to generate the full devfile including the "development" part.